keysign/KeyPresent.py

```diff
@@ -92,25 +91,32 @@ def __init__(self, key, qrcodedata=None, builder=None):
         self.uids_label = self._builder.get_object("uidsLabel")
         self.fingerprint_label = self._builder.get_object("keyFingerprintLabel")
         self.qrcode_frame = self._builder.get_object("qrcode_frame")
+        self.internet_switch = self._builder.get_object("internet_switch")
+        self.internet_spinner = self._builder.get_object("internet_spinner")
 
         self.key_id_label.set_markup(
             format_fingerprint(key.fingerprint).replace('\n', '  '))
         self.uids_label.set_markup("\n".join(
-                                        [GLib.markup_escape_text("{}".format(uid))
+                                        [glib_markup_escape_rencoded_text(
+                                            uid.uid)
                                         for uid
                                         in key.uidslist]))
         self.fingerprint_label.set_markup(format_fingerprint(key.fingerprint))
+        self.key_fingerprint = key.fingerprint
+
+    def set_fingerprint_code(self, code):
+        self.fingerprint_label.set_markup(code)
+
+    def set_qrcode(self, qrcodedata=None):
```

Comment:
With 32441eb commit I moved the initialization of kpw when we have both the code and the qr. This change allowed me to remove these methods.